trio/_core/_unbounded_queue.py

```diff
@@ -128,22 +148,14 @@ async def get_batch(self):
             finally:
                 await _core.cancel_shielded_checkpoint()
 
-    def statistics(self):
-        """Return an object containing debugging information.
-
-        Currently the following fields are defined:
-
-        * ``qsize``: The number of items currently in the queue.
-        * ``tasks_waiting``: The number of tasks blocked on this queue's
-          :meth:`get_batch` method.
-
-        """
-        return _UnboundedQueueStats(
+    def statistics(self) -> UnboundedQueueStatistics:
+        """Return an UnboundedQueueStatistics object containing debugging information."""
```

Comment:
```suggestion
        """Return an `UnboundedQueueStatistics` object containing debugging information."""
```

... maybe that's nicer?

Reply:
Or ```""":class:`UnboundedQueueStatistics`"""```, though as it's deprecated and not documented anywhere it doesn't matter all that much.